terraform/scripts/state_to_ansible_inventory.py

```diff
@@ -47,12 +47,17 @@
     print('{} ansible_host={}{}'.format(host['name'], host['ip'], master))
 print("[workers]")
 for host in inventory["worker"]:
-    print('{} ansible_host={}'.format(host['name'], host['ip']))
+    if "worker-0" in host['name']:
+        worker_quick = ' worker_type=quick'
+    else:
+        worker_quick = ''
+    print('{} ansible_host={}{}'.format(host['name'], host['ip'], worker_quick))
```

Comment:
this is kind of a hack. now that we seem to have solved the performance issues, maybe there is no need for this separation among workers anymore?

Reply:
hum, I still think it's a good thing to have a separate queue for fetch tasks due to some test jobs that still takes lots of time to process, e.g. android-lkft. I was thinking about going back to two workers only: one for fetching and another one for anything else...